priv/repo/migrations/20161128200526_create_user.exs

```diff
@@ -0,0 +1,12 @@
+defmodule MovieWagerApi.Repo.Migrations.CreateUser do
+  use Ecto.Migration
+
+  def change do
+    create table(:users) do
+      add :username, :string, null: false
```

Comment:
@SViccari will I need to add that to the migration as well? Should this also be a DB level constraint?